extra-x11/nvidia-prime/autobuild/overrides/usr/share/X11/xorg.conf.d/20-nvidia-prime.conf

```diff
@@ -0,0 +1,4 @@
+Section "ServerLayout"
+    Identifier "layout"
+    Option "AllowNVIDIAGPUScreens"
+EndSection
```

Comment:
Should this file be modified by users? If so, add `/usr/share/X11/xorg.conf.d/20-nvidia-prime.conf` to `autobuild/conffiles`.

Reply:
I think this file should not be modified by users.